recipes/keras-tuner/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "keras-tuner" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f018ded5127e7c588bd78faa753fa1f4108f1f28c0e5694a06bfb1f08204215b
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    # - tensorflow ==2.0
+  run:
+    - colorama
+    - future
+    - numpy
+    - python >=3.6
+    - requests
+    - scikit-learn
+    - scipy
+    - tabulate
+    - terminaltables
+    - tqdm
+    - tensorflow ==2.0
```

Comment:
Can this be less tightly pinned?

Reply:
it has to be at least 2.0, testing if it can be >=. the repo says its for tf version 2.0.

Reply:
@scopatz passes with the less strict pin.

Reply:
Great!